### PR TITLE
Improve Stack documentation.

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @j9brown, @abarth

**Comment by abarth** [comment]
LGTM

File: packages/flutter/lib/src/rendering/stack.dart [hunk 1]
```diff
@@ -419,21 +418,21 @@ abstract class RenderStackBase extends RenderBox
...
-/// space in the non-fixed dimensions.
+/// by subtracting the sum of the top and bottom values from the height of the stack.
+/// Similarly, if the child has right and left values that are both non-null,
+/// the child is given a fixed width derived from the stack's width.
+/// Otherwise, the child is given unbounded constraints in the non-fixed dimensions.
 ///
 /// Once the child is laid out, the stack positions the child according to the
-/// top, right, bottom, and left offsets. For example, if the top value is 10.0,
-/// the top edge of the child will be placed 10.0 pixels from the top edge of
+/// top, right, bottom, and left offsets. For example, if the bottom value is 10.0,
```
Comment (j9brown) [inline packages/flutter/lib/src/rendering/stack.dart]: Consider replacing the word "offsets" with "insets".

**Comment by Hixie** [comment]
I removed the word "offset" from that sentence.

**Pull request merged**